aviary/mission/flops_based/ode/mission_ODE.py

```diff
@@ -42,6 +39,10 @@ def initialize(self):
         self.options.declare(
             'use_actual_takeoff_mass', default=False,
             desc='flag to use actual takeoff mass in the climb phase, otherwise assume 100 kg fuel burn')
+        self.options.declare(
```

Comment:
add `values` argument to define what options are acceptable

Reply:
done!